api/manifest/manifest.go

```diff
@@ -1448,6 +1448,20 @@ func initApplication(dir string) error {
 		return err
 	}
 
+	if kind == "rails" {
```

Comment:
If we have language-specific things they should be handled in the language Dockerfiles. bin/web and initializers/convox.rb come from convox/rails. .dockerignore should probably be global to all languages.